src/global.css

```
@@ -11,6 +11,7 @@
   --color-secondary: #ff8473;
   --color-inactive: #fff8ee;
   --color-contrast: #fff;
+  --color-primary-light: #eff7ee;
```

Could you think about a more semantic name?

```suggestion
  --background-card: #eff7ee;
```

```suggestion
  --background-card: #eff7ee;
```